woocommerce/woocommerce-gutenberg-products-block
# Add Spanish translation files

As of this morning the Spanish translation of the plugin is complete enough to publish, so I've added it into the plugin like we have Danish and French (see #339). 

### How to test the changes in this Pull Request:

1. Set your site to Spanish
2. Try adding blocks, all block strings should be in Spanish.
